README.md

```diff
@@ -1,6 +1,6 @@
-# BetterTTV
+# <img src="https://raw.githubusercontent.com/night/betterttv/master/.github/bttvlogotransparent.gif" height="50"> BetterTTV
```

Comment:
```suggestion
# <img src=".github/bttvlogotransparent.gif" height="50"> BetterTTV
```
This works as a cleaner approach, though I'm unsure if `.github` is the best place to drop this

Reply:
thanks by the way :bowtie: